packages/office-ui-fabric-react/src/components/FloatingPicker/BaseFloatingPicker.types.ts

```diff
@@ -93,23 +93,24 @@ export interface IBaseFloatingPickerProps<T> extends React.ClassAttributes<any>
    */
   onRemoveSuggestion?: (item: IPersonaProps) => void;
   /**
-   * A function used to validate if raw text entered into the well can be added
+   * Determines if the input string should be resolved to an item in the well.
+   *
+   * Called only if the suggestion list is empty and we are not focused on a
+   * focusable header / footer item.
+   *
+   * If unset, all insertions of unmatched content will not be inserted.
    */
-  onValidateInput?: (input: string) => boolean;
-  /**
-   * The text to display while searching for more results in a limited suggestions list
-   */
-  searchingText?: ((props: { input: string }) => string) | string;
+  isQueryForceResolveable?: (inputQuery: string) => boolean;
 
   /**
-   * Function that specifies how arbitrary text entered into the well is handled.
+   * Function that specifies how a force resolved suggestion is entered into the well.
    */
-  createGenericItem?: (input: string, isValid: boolean) => ISuggestionModel<T>;
+  createForceResolvedItem?: (input: string) => T;
```

Comment:
This could be widened to T | ISuggestionsItem<T> in the current implementation, which could be useful if we moved ValidationState to ISuggestionsItem<>. Thoughts?